lua/avante/utils/init.lua

```diff
@@ -514,4 +516,32 @@ function M.debounce(func, delay)
   end
 end
 
+function M.get_project_root()
+  local Root = require("avante.utils.root")
+  return Root.git() or Root.cwd()
```

Comment:
```suggestion
	return M.root.get()
```

This should concat from lsp -> git -> cwd